crates/biome_cli/tests/snapshots/main_commands_check/apply_bogus_argument.snap

```diff
@@ -43,7 +43,7 @@ fix.js lint ━━━━━━━━━━━━━━━━━━━━━━
 ```block
 fix.js format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  × Format with errors is disabled.
+  × Code formatting aborted due to parsing errors. To format code with errors, enable the 'formatter.formatWithErrors' option.
```

Comment:
This would point to a regression, I think. I didn’t yet spot what might’ve caused it though.